src/Categories/Bicategory/Construction/LaxSlice.agda

```diff
@@ -0,0 +1,337 @@
+{-# OPTIONS --without-K --safe #-}
+
+-- Mentioned in passing https://ncatlab.org/nlab/show/slice+2-category
+
+open import Categories.Bicategory
+
+module Categories.Bicategory.Construction.LaxSlice
+       {o ℓ e t}
+       (𝒞 : Bicategory o ℓ e t)
+       where
+
+open import Categories.Enriched.Category
+open import Categories.Category renaming (Category to 1Category)
+open import Categories.Morphism.Reasoning
+
+open import Categories.Bicategory.Extras 𝒞
+
+open import Level
+
+record SliceObj (X : Obj) : Set (t ⊔ o) where
+  constructor sliceobj
+  field
+    {Y} : Obj
+    arr : Y ⇒₁ X
+
+module SliceHom (A : Obj) where
+
+  record Slice⇒₁ (X Y : SliceObj A) : Set (o ⊔ ℓ) where
+    constructor slicearr₁
+    private
+      module X = SliceObj X
+      module Y = SliceObj Y
+
+    field
+      {h} : X.Y ⇒₁ Y.Y
+      Δ   : X.arr ⇒₂ (Y.arr ∘ₕ h)
+
+  record Slice⇒₂ {X Y : SliceObj A} (H K : Slice⇒₁ X Y) : Set (ℓ ⊔ e) where
+    constructor slicearr₂
+    private
+      module Y = SliceObj Y
+      module H = Slice⇒₁ H
+      module K = Slice⇒₁ K
+
+    field
+      {ϕ} : H.h ⇒₂ K.h
+      E   : K.Δ ≈ (Y.arr ▷ ϕ ∘ᵥ H.Δ)
+
+  open hom.Equiv
```

Comment:
One reason for explicit `using` is things like this: took me forever to figure out (in the github editor, not emacs) where `hom` came from.